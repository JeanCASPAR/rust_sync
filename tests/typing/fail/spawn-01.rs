use rust_sync::sync;

sync! {
    #![pass(1)]

    node oui(a : int) = ()
    where
        b : int = spawn non(a, 0),
}

fn main() {}
