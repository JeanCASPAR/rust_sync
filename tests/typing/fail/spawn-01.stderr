error: unexpected end of input, expected identifier
 --> tests/typing/fail/spawn-01.rs:3:1
  |
3 | / sync! {
4 | |     #![pass(1)]
5 | |
6 | |     node oui(a : int) = ()
7 | |     where
8 | |         b : int = spawn non(a, 0),
9 | | }
  | |_^
  |
  = note: this error originates in the macro `sync` (in Nightly builds, run with -Z macro-backtrace for more info)
